src/mondo_mapping_status.py

```diff
@@ -0,0 +1,115 @@
+"""Mapping status between Medgen and Mondo"""
+from pathlib import Path
+from typing import List, Set, Tuple
+
+import pandas as pd
+
+SRC_DIR = Path(__file__).parent
+PROJECT_DIR = SRC_DIR.parent
+OUTDIR = PROJECT_DIR / 'output'
+RELEASE_OUTDIR = OUTDIR / 'release'
+INPUT_DIR = PROJECT_DIR / 'tmp' / 'input'
+MONDO_SSSOM_TSV = INPUT_DIR / 'mondo.sssom.tsv'
+MEDGEN_SSSOM_TSV = RELEASE_OUTDIR / 'medgen.sssom.tsv'
+# MEDGEN_PREFIXES: Some of these are old, some are new, some may not be used.
+MEDGEN_PREFIXES = ['Medgen', 'MEDGEN', 'Medgen_UID', 'UMLS', 'UMLS_CUI']
+CURIE = str
+
+
+def ids_prefixless(ids: Set[str]) -> Set[str]:
+    """Remove prefix"""
+    return set([x.split(':')[1] for x in ids])
+
+
+def ids_drop_uids(ids: Set[CURIE]) -> Set[CURIE]:
+    """From a set of Medgen IDs, drop those that are UIDs"""
+    return set([x for x in ids if x.split(':')[1].startswith('C')])
+
+def read_mapping_sources(
+    mondo_predicate_filter: List[str] = None,
+    drop_uids=True
+) -> Tuple[Set[CURIE], Set[CURIE], Set[CURIE]]:
+    """Read data sources
+    :param drop_uids: drop UIDs from Medgen IDs. These are ones that don't start with CN or C, and are IDs that are used
+    only internally in Medgen and are not stable."""
+    medgen_df = pd.read_csv(MEDGEN_SSSOM_TSV, sep='\t', comment='#').fillna('')
+    medgen_in_medgen: Set[CURIE] = set(list(medgen_df['subject_id']))
+
+    mondo_df = pd.read_csv(MONDO_SSSOM_TSV, sep='\t', comment='#').fillna('')  # n=72,902
+    mondo_df['prefix'] = mondo_df['object_id'].apply(lambda x: x.split(':')[0])
+    mondo_df = mondo_df[mondo_df['prefix'].isin(MEDGEN_PREFIXES)]  # n=16,627
+    del mondo_df['prefix']
+    # preds = list(mondo_df['predicate_id'].unique())  # only skos:exactMatch
```

Comment:
### exactMatch thoughts
I imagined we want to do the same thing here as with GARD where we only care about exact matches.
However a couple things I discovered so far are: 
- Our prior Mondo->Medgen mappings were only of `skos:exactMatch`
- All of the mappings coming out of Chris' pipeline are only of: `oboInOwl:hasDbXref` or `owl:equivalentClass`

Reply:
Replace all `owl:equivalentClass` to `skos:exactMatch` in the Medgen ingest. owl:equivalentClass is no longer relevant anywhere across our pipelines.

Reply:
I believe I've figured out how to do this. In OBO, this:
`equivalent_to: <CURIE>`
Should be changed to this:
`property_value: exactMatch <CURIE>`

I just need to change the Perl that creates the OBO like this. Then the pipeline will generate the correct OWL.